custom/icds_reports/tests/agg_tests/test_export_data.py

```diff
@@ -3366,8 +3366,8 @@ def test_growth_tracker_export(self):
                       None,
                       None,
                       'N/A',
-                      0,
-                      0,
+                      None,
```

Comment:
Why do these go from 0 to `None` and what does that change in the output?

Reply:
If the beneficiary is not pse_eligible we put his/her lunch count as NULL which is not present earlier.